types/semantic-release/lib/get-git-auth-url.d.ts

```diff
@@ -0,0 +1,7 @@
+import { Context } from "../index";
+
+declare function getGitAuthUrl(
+    context: Context,
+): Promise<string>;
+
+export = getGitAuthUrl;
```

Comment:
the current version seems to be full ESM module, weith 'export default' used:
https://github.com/semantic-release/semantic-release/blob/master/lib/get-git-auth-url.js#L62
the v17, is CJS based version